public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -494,28 +494,24 @@ export const AnomalyDetailsChart = React.memo(
                     //TODO: research more why only set this old property will work.
                     showLegendDisplayValue={false}
                     legendPosition={Position.Right}
-                    onBrushEnd={
-                      props.isHCDetector
-                        ? undefined
-                        : (brushArea: XYBrushArea) => {
-                            const start = get(
-                              brushArea,
-                              'x.0',
-                              DEFAULT_DATE_PICKER_RANGE.start
-                            );
-                            const end = get(
-                              brushArea,
-                              'x.1',
-                              DEFAULT_DATE_PICKER_RANGE.start
-                            );
-                            !props.bucketizedAnomalies
-                              ? handleZoomRangeChange(start, end)
-                              : handleDateRangeChange(start, end);
-                            if (props.onDatePickerRangeChange) {
-                              props.onDatePickerRangeChange(start, end);
-                            }
-                          }
-                    }
+                    onBrushEnd={(brushArea: XYBrushArea) => {
```

Comment:
For single-entity detector, the zooming will dynamically sample anomaly results and show. For example, if the date range is big and contains too many data points, we will sample 1000 anomaly results points and show them. For HC detector, is it the same logic?

Reply:
Yes, we bucketize the results the same way as the single entity. It is abstracted such that the results are handled without context of single entity vs. multi entity, real-time vs. historical, etc.